mappings/net/minecraft/block/entity/SkullBlockEntity.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_5291 net/minecraft/block/entity/SkullBlockEntity
+	FIELD field_25188 skullType I
```

Comment:
`type` ?

Reply:
no

Reply:
why? so u can copy mcp?

Reply:
what? no

Reply:
the field's type is `SkullType`

Reply:
its an int

Reply:
Have a look at `SkullBlock`. They dont call their enum as SkullType..its just `Type` the same applies here

Reply:
okay then